newIDE/app/src/stories/index.js

```diff
@@ -4270,3 +4275,74 @@ storiesOf('NewObjectDialog', module)
       onChoose={action('choose')}
     />
   ));
+
+storiesOf('CommandPalette', module)
+  .addDecorator(muiDecorator)
+  .add('commands', () => (
+    <I18n>
+      {({ i18n }) => (
+        <AutocompletePicker
+          i18n={i18n}
+          items={[
+            {
+              name: 'OPEN_PROJECT',
+              displayText: t`Open project`,
+              enabled: true,
+              handler: () => {},
+            },
+            {
+              name: 'SAVE_PROJECT',
+              displayText: t`Save project`,
+              enabled: true,
+              handler: () => {},
+            },
+            {
+              name: 'EDIT_OBJECT',
+              displayText: t`Edit object...`,
+              enabled: true,
+              handler: () => {},
+            },
+          ]}
+          onClose={() => {}}
+          onSelect={(item: NamedCommand) =>
+            action(`Open command ${item.name}`)()
+          }
```

Comment:
nit: you could even do just `onSelect={action("Open command")}` because the argument will be logged by Storybook :)

Reply:
Oh okay, that's nice :D. Though, I had to add a type annotation to the items array, otherwise Flow wasn't able to infer whether the array was one of commands or command options.